hw/ip/spi_host/data/spi_host.hjson

```diff
@@ -0,0 +1,590 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "pwm",
+  clock_primary: "clk_i",
+  other_clock_list: [ "clk_core_i" ],
+  reset_primary: "rst_ni",
+  other_reset_list: [ "rst_core_ni" ],
+  bus_device: "tlul",
+  bus_host: "none",
+  regwidth: "32",
+  param_list: [
+    { name: "ByteOrder",
+      desc: '''Byte order to use when transmitting or receiving data.  If ByteOrder = 0,
```

Comment:
i kind of wonder if software would prefer this as a configuration option and not a compile time option.

Reply:
@gkelly, @a-pronin any opinions?

Reply:
@eunchan 
ah i remember why i thought this now. In `spi_device`, the ordering is actually a configuration option.

Reply:
Ah, I think that totally makes sense for SPI_DEVICE.  If we weren't focusing *mostly* on flash devices I would say could be a useful run-time option for SPI_HOST.  If we shave a bit of the CONFIG.CLKDIV register we could fit it in CONFIG (it's full now with changes to CSNLEAD, TRAIL and IDLE.

Reply:
[Moving this comment](https://github.com/lowRISC/opentitan/pull/5149/files#r572618166) and all other related to byte-ordering and packing to #5149

Reply:
[Conversation carried over to #5149](https://github.com/lowRISC/opentitan/pull/5149/files#r572618166)